cli_test.go

```diff
@@ -53,3 +53,49 @@ func TestBasicCleanup(t *testing.T) {
 
 	testutil.FinishTest(t, dir, s1, s2, 200, 200)
 }
+
+func TestBasicConvertBadger2(t *testing.T) {
```

Comment:
As I mentioned in https://github.com/ipfs/ipfs-ds-convert/pull/23#discussion_r495199449 there aren't really any natural places to add Badger2 tests in the repo that are just copy-paste's of the Badger1 tests (aside from the short Validator tests). This is mostly because this library is mostly datastore agnostic and so it shouldn't be relevant.

The existing tests are:
1) The CLI works (cli_test.go)
2) The strategies work (strategy_test.go)
3) Datastore configs look right (validate_test.go)

I think the right way to make this work is to just add a basic test that converts from all primitive datastores (flatfs, leveldb, memory, badger, badger2) to in memory leveldb and then back again.

You should be able to use t.Run to create the various subtests and utilize the testutils (e.g. testutil.PrepareTest and FinishTest) to generate the random keys and check their validity.

Reply:
Done, with the exception of memory datastore.  If there is a valid memory datastore config example, then I will incorporate that in the tests.